packages/tocco-ui/src/EditableValue/typeEditors/SingleSelect.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import Select from 'react-select'
+
+const SingleSelect = props => {
+  const onChange = e => {
+    props.onChange(e)
+  }
+
+  return (
+    <Select
+      single
+      clearable
+      placeholder=""
+      noResultsText="-"
+      value={props.value}
+      onChange={onChange}
+      options={props.options.store}
+    />
+  )
+}
+
+SingleSelect.propTypes = {
+  onChange: React.PropTypes.func,
+  value: React.PropTypes.string,
+  options: React.PropTypes.shape({
+    store: React.PropTypes.arrayOf(
```

Comment:
In MultiSelect this is called "possibleValues" and here "store" please rename one or the other